pkg/overlay/cache.go

```diff
@@ -69,6 +71,16 @@ func (cache *Cache) Inspect(ctx context.Context) (storage.Keys, error) {
 	return nil, errors.New("not implemented")
 }
 
+// List returns a list of nodes from the cache DB
+func (cache *Cache) List(ctx context.Context, cursor storj.NodeID, limit int) ([]*pb.Node, error) {
+	return cache.db.List(ctx, cursor, limit)
+}
+
+// Paginate returns a list of `limit` nodes starting from `start` offset.
+func (cache *Cache) Paginate(ctx context.Context, limit int, offset int64) ([]*pb.Node, bool, error) {
```

Comment:
Usually the order is "offset, limit"

Reply:
I could change this no problem. I just followed the function signature of the generated dbx functions.